target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -220,6 +220,14 @@ eap300v2)
 	ucidef_set_led_netdev "lan" "LAN" "engenius:blue:lan" "eth0"
 	ucidef_set_led_wlan "wlan" "WLAN" "engenius:blue:wlan" "phy0tpt"
 	;;
+ens202ext)
+	ucidef_set_led_power "power" "POWER AMBER" "engenius:amber:power" "1"
```

Comment:
good point. the rssileds package is missing. please add DEVICE_PACKAGES containing the rssileds  package to your image build,